nessie/externals/redshift.py

```diff
@@ -76,8 +76,13 @@ def drop_external_schema(schema_name):
 
 
 def fetch(sql, **kwargs):
-    """Execute SQL read operation with optional keyword arguments for formatting, returning an array of named tuples."""
-    return _execute(sql, 'read', **kwargs)
+    """Execute SQL read operation with optional keyword arguments for formatting, returning an array of dictionaries."""
+    rows = _execute(sql, 'read', **kwargs)
+    if rows is None:
+        return None
+    else:
+        # For Pandas compatibility, copy psycopg's list-like object of dict-like objects to a real list of dicts.
+        return [r.copy() for r in rows]
```

Comment:
You'll notice many code diffs of the `results.count`-to-`results['count']` sort. This little requirement is the reason.